CHANGELOG.md

```diff
@@ -4,6 +4,11 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [1.0.3] - Unreleased
+### Fixed
+- [astarte_appengine_api] Property unset succeeds independently of whether there exist a device
```

Comment:
Property unset succeeds [...] -> Make property unset succeed [...]

Otherwise it's not clear whether you're describing the previous behaviour that was wrong or the new one that is right